src/modules/configurator/const/toppingArray.ts

```diff
@@ -0,0 +1,32 @@
+import { ToppingModel } from '../models';
+
+export const toppingArray: ToppingModel[] = [
+  {
+    title: 'Chilli',
+    id: 1,
+  },
+  {
+    title: 'Corn',
+    id: 2,
+  },
+  {
+    title: 'Egg',
+    id: 3,
+  },
+  {
+    title: 'Pineapple',
+    id: 4,
+  },
+  {
+    title: 'Meat',
+    id: 5,
+  },
+  {
+    title: 'Shrooms',
+    id: 6,
+  },
+  {
+    title: 'Bacon',
+    id: 7,
```

Comment:
možeš na svaki od njih dodati i `image: require("blabla")` da bi mogao napraviti onaj mapping što sam iznad objasnio